config.ru

```diff
@@ -28,4 +28,12 @@ configure do
   # end
 end
 
-Rack::Handler::Puma.run(TransportAPI, Port: config.port)
+host = "0.0.0.0"
+bind_addr = "ssl://#{host}?"
```

Comment:
These don't go in the configure block?

Reply:
Yeah :(  https://github.com/puma/puma#binding-tcp--sockets for that, and https://github.com/puma/puma/blob/master/lib/puma/dsl.rb for all config options (of which ssl is not there).